requirements/deploy.txt

```diff
@@ -292,7 +292,7 @@ pyjwt[crypto]==2.6.0
     # via
     #   -r requirements/pip.txt
     #   django-allauth
-pyquery==1.4.3
+pyquery==2.0.0
```

Comment:
I understand the new changes don't affect us: https://pyquery.readthedocs.io/en/latest/changes.html#id1 👍🏼